provider/internal/builtin/service_client.go

```diff
@@ -51,6 +53,12 @@ func (p *builtinServiceClient) Evaluate(ctx context.Context, cap string, conditi
 	log := p.log.WithValues("ruleID", cond.ProviderContext.RuleID)
 	log.V(5).Info("builtin condition context", "condition", cond, "provider context", cond.ProviderContext)
 	response := provider.ProviderEvaluateResponse{Matched: false}
+
+	template := engine.ChainTemplate{}
+	template.ExcludedPaths = append(template.ExcludedPaths, p.excludedDirs...)
+
+	cond.ProviderContext.Template[engine.TemplateContextPathScopeKey] = template
```

Comment:
you will need to merge it into the existing stuff not replace it

Reply:
Makes sense. Will update